### PR TITLE
docs: Settlement report blob storage to C4 diagram.

### PR DESCRIPTION
**Pull request by defectiveAi**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)
- [x] Documentation has been updated
- [ ] The integration [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) has been updated
- [x] C4 diagrams have been updated and Team Outlaws has been informed about relevant changes

### CODE REVIEW

File: docs/diagrams/c4-model/model.dsl [hunk 1]
```diff
@@ -6,6 +6,11 @@ wholesaleSubsystem = group "Wholesale" {
         technology "Azure Data Lake Gen 2"
         tags "Data Storage" "Microsoft Azure - Data Lake Store Gen1" "Mandalorian"
     }
+    wholesaleBlobStorage = container "Settlement Report Blob Storage" {
+        description "Contains (drafts of) settlement reports"
+        technology "Azure Blob Storage"
+        tags "Data Storage" "Raccoons"
```
Comment (dstenroejl) [inline docs/diagrams/c4-model/model.dsl:12]: Not sure if Racoons should be responsible?

**Review by dstenroejl — APPROVED** [review]
Approved, but with a comment

File: docs/diagrams/c4-model/model.dsl (hunk 1 above)
Comment (BjarkeMeier) [reply docs/diagrams/c4-model/model.dsl:12]: Is it a separate storage account / container / folder?

File: docs/diagrams/c4-model/model.dsl [hunk 2]
```diff
@@ -6,6 +6,11 @@ wholesaleSubsystem = group "Wholesale" {
         technology "Azure Data Lake Gen 2"
         tags "Data Storage" "Microsoft Azure - Data Lake Store Gen1" "Mandalorian"
     }
+    wholesaleBlobStorage = container "Settlement Report Blob Storage" {
+        description "Contains (drafts of) settlement reports"
```
Comment (BjarkeMeier) [inline docs/diagrams/c4-model/model.dsl:10]: Why drafts?

File: docs/diagrams/c4-model/model.dsl (hunk 1 above)
Comment (defectiveAi) [reply docs/diagrams/c4-model/model.dsl:12]: Yes, separate storage account.

File: docs/diagrams/c4-model/model.dsl (hunk 2 above)
Comment (defectiveAi) [reply docs/diagrams/c4-model/model.dsl:10]: Not "drafts" as a business term, but from a technical perspective - the storage also functions as a scratch storage and contains partially generated reports, until they are fully generated.

File: docs/diagrams/c4-model/model.dsl (hunk 1 above)
Comment (BjarkeMeier) [reply docs/diagrams/c4-model/model.dsl:12]: Good choice. And then I'm fine with Raccoons being responsible.

**Pull request merged**